repositories.bzl

```diff
@@ -257,3 +271,10 @@ def io_opencensus_grpc_metrics():
       artifact = "io.opencensus:opencensus-contrib-grpc-metrics:0.8.0",
       sha1 = "5e54d0e6dd946fe097e63ad68243e0006fbb1fbc",
   )
+
+def junit():
+  native.maven_jar(
+      name = "junit",
```

Comment:
To follow the [Bazel convention](https://docs.bazel.build/versions/master/be/workspace.html#maven_jar), this should be "junit_junit". And let's use the same name for the omit_* variable.

Ditto for Truth. Should be com_google_truth_truth.